pages/glossary/large-scale-text.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Large Scale Text
+key: large-scale-text
+---
+
+Text nodes with an [advance height](https://www.w3.org/TR/css-values/#length-advance-measure) of:
```

Comment:
Same comment regarding the definition of advance height.

Reply:
Same response :)